.travis.yml

```diff
@@ -13,12 +14,7 @@ script:
   - cargo build --verbose
   - cargo test --verbose
   - cargo clippy --all-targets --all-features -- -D warnings
-matrix:
-  include:
-    - rust: nightly
-      before_script:
-        - rustup component add rustfmt
-        # Fallback to git in case clippy is not available in the current nightly
-        - rustup component add clippy --toolchain=nightly || cargo install --git https://github.com/rust-lang/rust-clippy/ --force clippy
```

Comment:
We should keep this because clippy is not always available on nightly, see [here](https://rust-lang.github.io/rustup-components-history/x86_64-unknown-linux-gnu.html), and [here](https://github.com/rust-lang/rust-clippy#travis-ci)